tagbot.py

```diff
@@ -0,0 +1,106 @@
+import gitlab
```

Comment:
Just some style stuff. 
According to our iSort rules `import gitlab` should come after the python stuff

```python
import json
import os
import re
import traceback

import gitlab


re_repo = re.compile("Repository:.*/(.*/.*)")
```